posthog/api/test/dashboards/__snapshots__/test_dashboard.ambr

```diff
@@ -954,6 +954,9 @@
               OR "posthog_dashboardtile"."insight_id" IS NULL)
          AND "posthog_dashboardtile"."dashboard_id" = 2
          AND "posthog_dashboardtile"."dashboard_id" = 2
+         AND NOT ("posthog_dashboard"."deleted"
+                  AND "posthog_dashboardtile"."deleted"
+                  AND "posthog_dashboardtile"."deleted" IS NOT NULL)
```

Comment:
interesting change... I guess it's only enforcing that we don't return deleted things so probably fine